docs/release-notes/8.9.asciidoc

```diff
@@ -1,6 +1,19 @@
 [[release-notes-header-8.9.0]]
 == 8.9
 
+[discrete]
+[[release-notes-8.9.2]]
+=== 8.9.2
+
+[discrete]
+[[bug-fixes-8.9.2]]
+==== Bug fixes
+
+* Fixes legacy showTopN popover actions ({pull}165099[#165099]).
```

Comment:
@PhilippeOberti @christineweng should this be release-noted for 8.9.2? It looks like a fix for the expandable alert details flyout, which is being introduced in 8.10.

cc: @semd

Reply:
Confirmed with @PhilippeOberti that this isn't specific to the new expandable flyout, so I'll be keeping it in unless directed otherwise.

Reply:
```suggestion
* Fixes a bug that stopped the additional inline actions from completing ({pull}165099[#165099]).
```